src/plugins/identity/loader.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+
+import {type PluginDeclaration} from "../../analysis/pluginDeclaration";
+import {declaration} from "./declaration";
+
+export interface Loader {
+  declaration(): PluginDeclaration;
```

Comment:
Hmm, why have a method that returns the declaration, rather than just having the declaration as a (read-only) property?

Note that having the function wrapper doesn't provide any additional safety because we are returning the same object either way, but it's fine because the object is deep-frozen by convention.

Reply:
No strong reason for this. Because in Flow the interfaces can do all sorts of wild things, it's almost the same as an object type. You can even turn them into object maps https://flow.org/en/docs/types/interfaces/#toc-interfaces-as-maps :scream: I find usages like these surprising for interfaces, so I arbitrarily stuck with just functions.